### PR TITLE
Add instructions of using automation scripts

### PR DESCRIPTION
**Pull request by boyuanzz**
r: @aaltay @pabloem

### CODE REVIEW

**Comment by aaltay** [comment]
@boyuanzz have you had a chance to look at the test failure?

**Comment by boyuanzz** [comment]
retest this please

**Comment by boyuanzz** [comment]
retest this please

**Comment by boyuanzz** [comment]
Tests passed. You can view the preview here: https://github.com/boyuanzz/beam-site/blob/new_release_guide/src/contribute/release-guide.md @aaltay

File: src/contribute/release-guide.md [hunk 1]
```diff
@@ -528,6 +646,36 @@ If there are no issues, reply on the vote thread to close the voting. Then, tall
 ### Run validation tests
 All tests listed in this [spreadsheet](https://s.apache.org/beam-release-validation)
 
+Since there are a bunch of tests, we recommend you running validatoins using automation script. In case of script failure, you can still run all of them manually.
```
Comment (aaltay) [inline src/contribute/release-guide.md]: validatoins -> validations

File: src/contribute/release-guide.md [hunk 2]
```diff
@@ -181,135 +201,207 @@ You should verify that the issues listed automatically by JIRA are appropriate t
...
+* Script: [start_snapshot_build.sh](https://github.com/apache/beam/blob/master/release/src/main/scripts/start_snapshot_build.sh)
 
-Check out the version of the codebase from which you start the release. For a new minor or major release, this may be `HEAD` of the `master` branch. To build a hotfix/incremental release, instead of the `master` branch, use the release tag of the release being patched. (Please make sure your cloned repository is up-to-date before starting.)
+* Usage
+  
+      ./beam/release/src/main/scripts/start_snapshot_build.sh
 
-    git checkout <master branch OR release tag>
+* Tasks included
+  1. Install [hub](https://github.com/github/hub) within your agreement.
```
Comment (aaltay) [inline src/contribute/release-guide.md]: within -> with

**Comment by boyuanzz** [comment]
Re @aaltay 
Commit https://github.com/apache/beam-site/pull/531/commits/0489b590f6b1d7a8c0c9732ff5694af18562817a addressed previous comments.

**Comment by aaltay** [comment]
@asfgit merge

**Pull request closed without merging**